src/main/java/com/esri/samples/localserver/local_server_map_image_layer/LocalServerMapImageLayerSample.java

```diff
@@ -75,7 +75,7 @@ public void start(Stage stage) throws Exception {
       server.addStatusChangedListener(status -> {
         if (status.getNewStatus() == LocalServerStatus.STARTED) {
           // start map image service
-          String mapServiceURL = new File("./samples-data/local_server/RelationshipID.mpk").getAbsolutePath();
+          String mapServiceURL = new File("./samples-data/local_server/USCitiesStates.mpk").getAbsolutePath();

```

Comment:
I updated this sample to use a different mpk because Relationship.mpk is not working as a Feature Service and I can't find any updated version of it.

Its Location, `\\apps-data\Data\api\LocalServer\100-Test-Data\MPK` will need to replace old mpk with new in samples-data